packages/web-frontend/src/utils/periodGranularities.js

```diff
@@ -115,6 +120,14 @@ export function roundStartEndDates(granularity, startDate = moment(), endDate =
   };
 }
 
+export const formatMomentAsString = (date, granularity, format) =>
```

Comment:
I find `{{typeA}}to{{typeB}}`/`{{formatA}}to{{formatB}}` names quite intuitive for this sort of functionality:

```suggestion
export const momentToDateString = (date, granularity, format) =>
```

Reply:
Me too now that I see it!